concepticondata/conceptlists/Gruehn-2008-200.tsv

```diff
@@ -0,0 +1,201 @@
+Gruehn-2008-200-171	171	traurig	sad	699	SAD	16.905	2.667	2.667	1.909	2.917	2.667	2.435	2.553	3.583	3.667	4.583	3.75	3.625	4.167	3.896	2.25	1.917	2.727	2.5	2.083	2.609	2.34	6.455	6.417	6.182	6.167	6.435	6.174	6.304	3.167	3.167	2.333	2.583	3.167	2.458	2.813	3.667	3.167	3.917	3.667	3.417	3.792	3.604	1.938	2.75	2
+Gruehn-2008-200-172	172	treu	loyal			10.535	6.583	5.833	6.833	6.583	6.208	6.708	6.458	2.25	3.167	2.417	3.083	2.708	2.75	2.729	4.167	5.167	4.75	4.75	4.667	4.75	4.708	3.917	2.75	4.333	4.167	3.333	4.25	3.792	6.25	5.333	6.167	5.917	5.792	6.042	5.917	3.583	3.75	4.167	3.917	3.667	4.042	3.854	5.438	6.188	1.083
+Gruehn-2008-200-173	173	ueberheblich	contemptuous			1.285	1.75	1.75	1.833	1.917	1.75	1.875	1.813	5.333	5.5	5.417	5.583	5.417	5.5	5.458	4.167	3.5	4.083	2.917	3.833	3.5	3.667	4.75	3.833	4.5	4.727	4.292	4.609	4.447	2.75	2.917	1.583	1.917	2.833	1.75	2.292	2.083	2.083	2.167	2.083	2.083	2.125	2.104	2	1.625	1.083
+Gruehn-2008-200-174	174	ueberrascht	surprised	1572	SURPRISED	34.34	5.083	4.667	4.417	4.25	4.875	4.333	4.604	4	4	4.167	4.917	4	4.542	4.271	3.333	2.25	2.818	3.25	2.792	3.043	2.915	6.167	5.833	5	4.545	6	4.783	5.404	3.667	3.417	3.167	3.167	3.542	3.167	3.354	3.167	3	3	3.25	3.083	3.125	3.104	5.063	4.688	1.958
+Gruehn-2008-200-175	175	umsichtig	prudent			911	5.667	5.333	6.667	6	5.5	6.333	5.917	3.583	3.5	3.167	3.833	3.542	3.5	3.521	5.083	5.75	5.583	5.417	5.417	5.5	5.458	2.1	3.833	4.5	3.667	3.417	4.083	3.75	5.333	5	6.25	5.833	5.167	6.042	5.604	3.333	3.5	3.833	3.75	3.417	3.792	3.604	5.438	5.688	1.333
+Gruehn-2008-200-176	176	unangenehm	unpleasant	26	UNPLEASANT	6.313	1.75	1.417	1.417	2.091	1.583	1.739	1.66	5.75	4.833	5.333	4.455	5.292	4.913	5.106	2.667	2.833	4.167	2.667	2.75	3.417	3.083	3.25	3.583	4.583	3.75	3.417	4.167	3.792	1.667	2.083	2	2.167	1.875	2.083	1.979	3	3.083	2.917	2.75	3.042	2.833	2.938	1.813	1.313	1
+Gruehn-2008-200-177	177	ungestuem	impetuous			1.9	3.75	3.5	3.167	3.417	3.625	3.292	3.458	3.917	5.5	5.417	5.167	4.708	5.292	5	2.917	2.583	3.5	3	2.75	3.25	3	5.083	4.583	5.667	5.417	4.833	5.542	5.188	3.083	3.333	2.5	2.5	3.208	2.5	2.854	2.25	1.417	1.583	1.75	1.833	1.667	1.75	3.563	3.5	1.083
+Gruehn-2008-200-178	178	unnahbar	unapproachable			1.2	2.833	2.5	2.667	2.583	2.667	2.625	2.646	5	4.583	5	4.5	4.792	4.75	4.771	4.583	3.75	4.417	3.917	4.167	4.167	4.167	4.182	3.333	4.75	4.5	3.739	4.625	4.191	2.833	3.333	2.25	3.417	3.083	2.833	2.958	2.917	2.833	3.25	3.5	2.875	3.375	3.125	2.438	2.25	1
+Gruehn-2008-200-179	179	unruhig	jittery			5.352	2.25	2.333	2.5	3	2.292	2.75	2.521	5.917	5.5	5.833	5.583	5.708	5.708	5.708	2.333	2.333	4.167	2.917	2.333	3.542	2.938	5	4.417	4.667	5.167	4.708	4.917	4.813	4.083	3.75	2.333	3.5	3.917	2.917	3.417	2.667	2.25	2.583	2.583	2.458	2.583	2.521	2.25	2.563	1.833
```

Comment:
Maybe `3975 NERVOUS`?

Reply:
We already have a mapping with "nervoes" so I'd leave this one unmapped.